lesson4_task3.py

```diff
@@ -0,0 +1,14 @@
+"""
+The name check:
+
+Write a program that has a variable with your name stored (in lowercase) and then asks for your name as input.
+The program should check if your input is equal to the stored name even if the given name has another case, e.g.,
+if your input is “Anton” and the stored name is “anton”, it should return True
+"""
+
+name = 'oleh'
+name_check = input('What is your name? ')
```

Comment:
Nicely done! 
You could also try to make this program continuously get an input from user, with while loop, and add `exit` command to stop infinite loop. Like:
```python
if name_check.lower() == 'exit':
    print('Exiting...')
    break
```